SO-98 Not Found Page

DGConn opened the pull request:
Not Found page

RealHoltz committed:
SO-48 Created Settings Page

-created components for the page and content

Miguel1357 committed:
SO-60

- Added send password reset email link
- Passwords can now be updated
- Component for recover password and update password
- Tested cases for password update errors

KiranKaur3 committed:
SO-62-Loader-Page-Web-Page-View

jvniorrr committed:
SO-68

- Create Organization card component.

jvniorrr committed:
SO-94

- SO-68 Creation of new organizations, and referencing only associated organizations
- Specific projects page template
- Member invites setup
- Member card component
- Project Attachment placeholder setup, needs connection to database and storage of images with URLS now
- SO-68 updates for deleting organizations dependent on membership association

jvniorrr committed:
SO-58

- New Project modal overlay
- Retrieving projects for members
- Minor changes to styles of other components

RealHoltz committed:
SO-48

- add some comments onto code

- helped component SettingsNav

JDoan03 committed:
SO-58

- create new projects functionality
- new project status enum
- New Project modal overlay
- Retrieving projects for members
- Minor changes to styles of other components
- Creation of new projects functionality
- new project status enum coinciding with database
- RLS updates on supabase as well, trigger function was DEFINER

KiranKaur3 committed:
SO-62
- minor fixes for loader

hashemJaber committed:
SO-91
- Added Tasks component
- Added Tasks edit component
- Added client apis to select, add, patch tasks
- Added Tasks CRU components
- Added Task members CRUD operations
- Added Validation between user and admin and setted up security policies on SupaBase

jvniorrr committed:
SO-92

- SO-92 and SO-68 timestamp component...
- Edit & Create Modal for timestamps
- Delete options for timestamp entries
- Error handling (60%)

RealHoltz committed:
SO-49

- Added server api calls for project member  tasks
- implemented code to pull member id related to specific project
- implemented retrieving tasks with specific member in project

Miguel1357 committed:
SO-77
- Added Create and Update API's to reciepts
- Updated Reciept interface from database.interface

hashemJaber committed:
SO-93
- Page for adding receipts provided by a  USER for a SPECIFIC PROJECT

-Page with OCR technogoligy that automates information extracting via Taggun API

-Page with editable table for information edit in case of OCR failure

JDoan03 committed:
SO-102
- Edit modal for projects
- Status buttons fix on edit projects modal

DGConn committed:
SO-98 Not Found Page

Just fixed the issues you asked (and it still works)

hashemJaber commented:
was this resolved ?
was is the code error ?

jvniorrr commented on src/app/not-found.tsx:
```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+import Buttons from '@/components/SharedComponents/Buttons';
+import { Lexend_Giga } from 'next/font/google';
+const inter = Lexend_Giga({ subsets: ['latin'] });
```
unneeded import, can be removed

jvniorrr commented on src/app/not-found.tsx:
```diff
@@ -0,0 +1,34 @@
+
+export default function NotFound() {
+	return (
+		<main className="flex min-h-screen flex-col items-center justify-between">
```
for this class name maybe use 
`<main className="flex min-h-screen flex-col items-center justify-between pt-5">`

jvniorrr commented on src/app/not-found.tsx:
```diff
@@ -0,0 +1,34 @@
+export default function NotFound() {
+	return (
+		<main className="flex min-h-screen flex-col items-center justify-between">
+			<section id="home-section">
```
`<section id="home-section flex flex-col items-center justify-between">`

jvniorrr reviewed (changes requested):
Solid work. Working as expected. Just few changes to be incorporated. 

Will also need to rebase your current branch to be synced / aligned with the main branch.

hashemJaber committed:
SO-74

- Apply Create operation via SupaBase for Reciepts
- Apply Read operation via SupaBase for Reciepts
- Apply Update operation via SupaBase for Reciepts
- Apply Delete operation via SupaBase for Reciepts

hashemJaber committed:
SO-93

SO-93

- Added Receipt item creation form
- Added Receipt item extractor
- Added Receipt item list form
- Added Page for adding receipts provided by a USER for a SPECIFIC PROJECT
- Added Page with OCR technology that automates information extracting via Taggun API
- Added Page with editable table for information edit in case of OCR failure

jvniorrr commented:
Separate PR created, and merged..
